node-wasm/src/node.rs

```diff
@@ -92,6 +93,10 @@ impl NodeDriver {
         worker_script_url: &str,
         worker_type: Option<NodeWorkerKind>,
     ) -> Result<NodeDriver> {
+        if let Err(e) = request_storage_persistence().await {
```

Comment:
since this only affects quota in ff, can we guard this to be only called in ff?

Reply:
I could have mentioned that beforehand, but

> In Firefox, when a site chooses to use persistent storage, the user is notified with a UI popup that their permission is requested.
> Safari and most Chromium-based browsers, such as Chrome or Edge, automatically approve or deny the request based on the user's history of interaction with the site and do not show any prompts to the user.

So the prompt would be displayed on firefox, other browser doing some heuristics. Maybe just leave it for all browsers then? We are intending on manually managing the data anyway and I want to do as little parsing of UserAgent as possible. Sniffing for chrome already felt bad enough :P